### PR TITLE
AWS-WAF Adding Modeling Rules

### PR DESCRIPTION
**Pull request by nkanon**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Description
This PR is created to add modeling rules for AWS WAF.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/AWS_WAF/README.md [hunk 1]
```diff
@@ -1,3 +1,38 @@
+<~XSIAM>
+# AWS WAF
+This pack includes Cortex XSIAM content.
+
+
+## Configuration on Server Side
+- For information on configuring **ACL web logging**, please refer to the following documentation [here](https://docs.aws.amazon.com/waf/latest/developerguide/logging-management.html).
```
Comment (ShirleyDenkberg) [inline Packs/AWS_WAF/README.md:7]:
```suggestion
- For information on configuring **ACL web logging**, refer to the following [documentation](https://docs.aws.amazon.com/waf/latest/developerguide/logging-management.html).
```

File: Packs/AWS_WAF/README.md [hunk 2]
```diff
@@ -1,3 +1,38 @@
...
+- For information on sending logs ACL web logging to **S3 bucket**, please refer to the following documentation [here](https://docs.aws.amazon.com/waf/latest/developerguide/logging-s3.html).
```
Comment (ShirleyDenkberg) [inline Packs/AWS_WAF/README.md:8]:
```suggestion
- For information on sending ACL web logs  to **S3 bucket**, refer to the following [documentation](https://docs.aws.amazon.com/waf/latest/developerguide/logging-s3.html).
```

File: Packs/AWS_WAF/README.md [hunk 3]
```diff
@@ -1,3 +1,38 @@
...
+| Name              | A descriptive name for your log collection configuration                                           | \<InstanceName\>    |
+| AWS Client ID     |  The Access key ID, which was received when configuring access keys for the AWS IAM user in AWS    | \<AWSClientID\>     |
+| AWS Client Secret | The Secret access key, which was received when configuring access keys for the AWS IAM user in AWS | \<AWSClientSecret\> |
+| Log Type          | Select Generic to configure your log collection to receive generic logs from Amazon S3             | Generic           |
+| Log Format        | Select the log format type as JSON                                                                 | Json              |
+| Vendor            | Set as 'aws'                                                                                       | aws               |
+| Product           | Set as 'waf'                                                                                       | waf               |
+| Compression       | Select 'gzip'                                                                                      | gzip              |
```
Comment (ShirleyDenkberg) [inline Packs/AWS_WAF/README.md:31]:
```suggestion
| SQS URL           | The ARN of the Amazon SQS that you configured in the AWS Management Console.                        | \<YourSQSURL\>      |
| Name              | A descriptive name for your log collection configuration.                                           | \<InstanceName\>    |
| AWS Client ID     |  The access key ID, which was received when configuring access keys for the AWS IAM user in AWS.    | \<AWSClientID\>     |
| AWS Client Secret | The secret access key, which was received when configuring access keys for the AWS IAM user in AWS. | \<AWSClientSecret\> |
| Log Type          | Select Generic to configure your log collection to receive generic logs from Amazon S3.             | Generic           |
| Log Format        | Select the log format type as JSON.                                                                 | Json              |
| Vendor            | Set as 'aws'.                                                                                      | aws               |
| Product           | Set as 'waf'.                                                                                       | waf               |
| Compression       | Select 'gzip'.                                                                                      | gzip              |
```

File: Packs/AWS_WAF/ReleaseNotes/1_0_2.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
+
+#### Modeling Rules
+
+##### New: AWS WAF Modeling Rule
+
+- Adding Modeling Rules for AWS WAF.
```
Comment (ShirleyDenkberg) [inline Packs/AWS_WAF/ReleaseNotes/1_0_2.md:6]:
```suggestion
Added Modeling Rules for AWS WAF.
```

File: Packs/AWS_WAF/ReleaseNotes/1_0_2.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+
+#### Parsing Rules
+
+##### New: AWS WAF Parsing Rule
+
+- Adding Parsing Rule for AWS WAF.
```
Comment (ShirleyDenkberg) [inline Packs/AWS_WAF/ReleaseNotes/1_0_2.md:12]:
```suggestion
Added Parsing Rule for AWS WAF.
```

**Comment by ShirleyDenkberg** [comment]
@nkanon Doc review completed.

**Review by eepstain — APPROVED** [review]

**Pull request merged**